utils/distarray_conda-env.yml

```diff
@@ -0,0 +1,44 @@
+name: distarray
+dependencies:
```

Comment:
Ok so it turns out the whole point of being able to specify environments with the `yml` file is that you can EXACTLY specify the environment. Conda does not resolve any dependencies when this route is taken. So when I removed all but the things I had installed at the command lines, conda didn't even install things like `setuptools`. 

However I see your point that if we stick to this methodology we will have to keep updating this `yml` file for the quick start to remain current. I will try to devise an alternative strategy.